Resources/Prototypes/Entities/Clothing/Eyes/NightVision.yml

```
@@ -0,0 +1,117 @@
+- type: entity
+  parent: ClothingEyesBase
+  id: ClothingEyesNVG
+  name: night vision goggles
+  description: Now you can see in the dark! It has the label "BL CORP technology".
+  components:
+    - type: Sprite
+      sprite: Clothing/Eyes/NVG/nightvision.rsi
+      layers:
+      - state: icon
+        map: [ "enum.ToggleVisuals.Layer" ]
+    - type: Clothing
+      sprite: Clothing/Eyes/NVG/nightvision.rsi
+    - type: NightVision # MUST HAVE ALL TINT!
```

```suggestion
        map: [ "enum.ToggleVisuals.Layer" ]
    - type: Clothing
      sprite: Clothing/Eyes/NVG/nightvision.rsi
    - type: NightVision
```
Surely not?... The component has default states for tints.